code/game/objects/items/devices/radio/headset.dm

```diff
@@ -18,6 +18,7 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	name = "radio headset"
 	desc = "An updated, modular intercom that fits over the head. Takes encryption keys."
 	icon_state = "headset"
+	worn_icon = "headset"
```

Comment:
This isn't quite that simple for headsets, sadly.

Reply:
> worn_icon is always an icon file. This isn't correct in the slightest.

So true